tests/test_apertures.py

```diff
@@ -46,15 +46,18 @@ def test_fgcmApertureHsc(self):
 
         butler = dafPersist.Butler(os.path.join(self.dataDir, 'hsc'))
 
-        schema = butler.get('src_schema').schema
+        dataRef = butler.dataRef('src', visit=34648, ccd=51)
+
+        self.assertRaises(RuntimeError, computeApertureRadiusFromDataRef, dataRef, 'base_PsfFlux_instFlux')
+        self.assertRaises(RuntimeError, computeApertureRadiusFromDataRef, dataRef, 'not_a_field')
+        self.assertEqual(computeApertureRadiusFromDataRef(dataRef, 'slot_CalibFlux_instFlux'), 12.0)
+        self.assertEqual(computeApertureRadiusFromDataRef(dataRef,
+                                                          'base_CircularApertureFlux_12_0_instFlux'),
+                         12.0)
+        self.assertEqual(computeApertureRadiusFromDataRef(dataRef,
+                                                          'base_CircularApertureFlux_4_5_instFlux'),
+                         4.5)
```

Comment:
This indentation hurts me, but it is correct, and is in fact informative, so I'm not going to complain about it even though I just did.